features/git-town/configuration/configuration_automatic.feature

```diff
@@ -9,10 +9,43 @@ Feature: Automatically running the configuration wizard if Git Town is unconfigu
     Given I haven't configured Git Town yet
 
 
-  Scenario Outline: Proceeding to configuration upon initial config prompt
+  Scenario Outline: Seeing a configuration prompt when running a Git Town command
+    When I run `<COMMAND>` and enter "^C"
+    Then I see the initial configuration prompt
+
+    Examples:
+    | COMMAND            |
+    | git extract        |
+    | git hack           |
+    | git kill           |
+    | git prune-branches |
+    | git pull-request   |
+    | git repo           |
+    | git ship           |
+    | git sync           |
+    | git sync-fork      |
+
+
+  Scenario Outline: Explicitly proceeding to configuration wizard upon seeing config prompt
```

Comment:
[This](https://github.com/Originate/git-town/blob/aw-prompt-default/features/git-town/configuration/configuration_automatic.feature#L64)?